Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1362,6 +1362,9 @@ ExceptionOr<void> HTMLElement::showPopover(const HTMLFormControlElement* invoker
 
     ASSERT(!isInTopLayer());
 
+    PopoverData::ScopedStartShowingOrHiding scopedWasShowingOrHiding(*this);
```

Comment:
Maybe we could rename this to `showOrHidingPopoverScope`.

Reply:
Done.